dts/bindings/gpio/nordic,nrf-gpio-forwarder.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This is an abstract device responsible for forwarding pins between cores.
+
+    Here is an example of how a nrf-gpio-forwarder can be used with a nRF21540
+    Front-End module.
+
+    nrf_radio_fem: nrf21540 {
```

Comment:
I would clarify that `nrf21540` node is required only in DT for Net Core, while `nrf-gpio-forwarder` is required only in DT for App Core. Now user can have an impression that both entries are required for both cores.